nodeEditor/src/blockTools.ts

```diff
@@ -124,7 +124,7 @@ export class BlockTools {
                 return new TextureBlock("Texture", nodeMaterial.mode === NodeMaterialModes.Particle);
             case "ReflectionTextureBlock":
                 return new ReflectionTextureBlock("Reflection texture");
-            case "LightBlock":
+            case "LightsBlock":
```

Comment:
why this change?

Reply:
@PirateJC asked me to change it.

Reply:
This could be misleading when people will found out that the block name is not the same

Reply:
More importantly it's a breaking change, all node materials that are currently using `LightBlock` will fail loading.

Reply:
Totally. I should have mentioned that also. I'm getting old

Reply:
OH. That stinks I will remove it then.